README.md

```diff
@@ -109,6 +109,55 @@ A glossary entry is structured like this:
     should `define('something')` in RStudio put the definition in the help pane
     (and if so, should it hyperlink to terms that the definition depends on)?
 
+## Use Cases
+
+1.  **Linking to a definition.**
+    1.  *Amari* writes a lesson in R Markdown that introduces some new terms.
+    1.  She has defined the language to be Spanish using the `glossary/language` key
+        in the YAML header,
+        but has not changed any other settings.
+    1.  She adds an inline code block `` `r gdef('linear-model', 'Linear models')` ``
+        to her lesson.
+    1.  When she knits her document,
+        the inline code block produces the HTML
+        `<a href="http://carpentries.org/glossary/es/#linear-model" class="glossary-definition">Linear Models</a>`
+
+1.  **Checking a lesson.**
+    1.  *Beatriz* has made some changes to a lesson she inherited from *Amari*,
+        and wants to check that it is still consistent.
+    1.  She runs a command-line script that:
+        1.  Reads the R Markdown file.
+        1.  Extracts the terms under the `glossary/defines` key.
+        1.  Searches the body of the document for calls to `gdef(...)`.
+        1.  Checks that every term listed in `glossary/defines` is referenced in the document body,
+            and that every term referenced in the document body is mentioned in `glossary/defines`.
+
+1.  **Finding lessons.**
+    1.  *Amari* writes a lesson in R Markdown.
+        She adds the `glossary` key to its YAML metadata
+        and indicates that the lesson requires the term `correlation`
+        and defines the term `regression`.
+    1.  *Beatriz* is writing a lesson on linear models.
+        She adds YAML metadata indicating that
+        the lesson requires the term `regression`.
+    1.  To find prerequisite lessons she can recommend to her students,
```

Comment:
I don't feel this should be a use case. It feels more natural for it to be a function which points to a folder and then we get all of the Rmds in that folder and run ourselves the rmarkdown function.

Reply:
Sorry, it should be a use case but wrapped under a function.

Reply:
:+1: there should be a function that can be called directly from R with a list of filenames. There should also be a command-line script that calls that function that can be integrated into builds.